Gemfile.lock

```diff
@@ -52,9 +52,6 @@ GEM
     image_optim_pack (0.6.0.20200215)
       fspath (>= 2.1, < 4)
       image_optim (~> 0.19)
-    image_optim_pack (0.6.0.20200215-x86_64-darwin)
-      fspath (>= 2.1, < 4)
-      image_optim (~> 0.19)
```

Comment:
@dshevtsov  -- is it OK to have this update in this PR? I usually exclude any changes to this file.

Reply:
no, `Gemfile.lock` should not be affected by content related pull requests.

Reply:
As to this exact change, it is already in `master` (https://github.com/magento/merchdocs/blob/master/Gemfile.lock#L55).
I'd expect it to be in this branch as well.

Reply:
This branch is up-to-date with master -> core-2.4.0-community. So should we just let it ride?

Reply:
Tried switching the bas and then back to see if it would re-synch the list of changes -- no go

Reply:
I created new commit to revert